### PR TITLE
[deutschebahn] Fix example and update links

### LINKED ISSUE #16577: Deutsche Bahn Binding - error "timetable.things"
Example file "timetable.things" 
#1 wrong configuration
Parameter "accessToken="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" is not part of acutal bridge configuration.
#2 defining "additionalFilter" is not working as parameter is defined by "filter vale" , e.g. departure#line="RE60" alöready includes ""

old "timetable.things"
Bridge deutschebahn:timetable:timetableLehrte "Fahrplan Lehrte" [ accessToken="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", trainFilter="departures", evaNo="8000226" ] {
  Thing deutschebahn:train:timetableLehrte:lehrteZug1 "Zug 1" [ position="1" ]
  Thing deutschebahn:train:timetableLehrte:lehrteZug2 "Zug 2" [ position="2" ]
}

## Possible Solution
Modified as follow and bridge is working
new "timetable.things" 
Bridge deutschebahn:timetable:timetableLehrte "Fahrplan Lehrte" [ clientSecret="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", clientId="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", trainFilter="departures", evaNo="8000226" ] {
  Thing deutschebahn:train:timetableLehrte:lehrteZug1 "Zug 1" [ position="1" ]
  Thing deutschebahn:train:timetableLehrte:lehrteZug2 "Zug 2" [ position="2" ]
}

## Your Environment
* Version used: openhab 4.1.2
* running docker with repository openhab/openhab:latest-debian
* Operating System UNRAID 6.12.8

### PR DESCRIPTION
**Pull request by holgerfriedrich**
Fixes #16577.
The example was not in sync with the documentation, credentials are provided differently.
Definition of Things no longer causes warnings.
Added hint to how to escape data in textual config.

Unfortunately, Deutsche Bahn does no longer provide [Station IDs](https://de.wikipedia.org/wiki/Interne_Bahnhofsnummer) needed for this binding (```evaNo```) in the Open Data portal. Although links to other portals were provided, I could not find the official list.
Thus, I added two links to other pages providing this information.

### CODE REVIEW

File: bundles/org.openhab.binding.deutschebahn/README.md [hunk 1]
```diff
@@ -29,8 +29,9 @@ To configure a timetable you first need to register at Deutsche Bahn developer p
 
 ### Determine the EVA-No of your station
 
-For the selection of the station within openHAB you need the eva no. of the station.
-You can look up the number within the csv file available at [Haltestellendaten](https://data.deutschebahn.com/dataset.tags.EVA-Nr..html).
+For the selection of the station within openHAB you need the eva no. of the station, i.e. a 7-digit number starting with 80 for Germany.
+The station ID is no longer available on the Open Data portal of Deutsche Bahn.
+You can look up the number at [here](https://www.michaeldittrich.de/ibnr/) or in the csv file [here](https://github.com/ratopi/haltestellendaten).
```
Comment (lsiepel) [inline bundles/org.openhab.binding.deutschebahn/README.md:34]:
```suggestion
You can look up the number at [www.michaeldittrich.de](https://www.michaeldittrich.de/ibnr/) or in [the csv file](https://github.com/ratopi/haltestellendaten).
```

**Review by lsiepel — COMMENTED** [review]
Minor comment for consideration

File: bundles/org.openhab.binding.deutschebahn/README.md (hunk 1 above)
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.deutschebahn/README.md]: done

[the remark this answers is not in the document]
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.deutschebahn/README.md]: done

**Pull request merged**